### PR TITLE
add downloads stats back to readme

### PR DESCRIPTION
**Pull request by olaoluwasalami**
@skzhang1  @sonichi @randombet 

## Related issue number

Closes #92

## Checks

- [x] I've included any doc changes needed for https://ag2ai.github.io/ag2/. See https://ag2ai.github.io/ag2/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,25 +23,27 @@
...
 > :fire: :tada: Nov 11, 2024: We are evolving AutoGen into AG2! A new organization [ag2ai](https://github.com/ag2ai) is created to host the development of AG2 and related projects with open governance. We invite collaborators from all organizations and individuals to join the development.
 
+:fire: :tada: AG2 is available via `ag2` (or its alias `autogen` or `pyautogen`) on PyPI! Starting with version 0.3.2, you can now install AG2 using:
 
-:fire: :tada: AG2 is available via `ag2` (or its alias  `autogen` or  `pyautogen`)  on PyPI! Starting with version 0.3.2, you can now install AG2 using:
 ```
 pip install ag2
 ```
+
 or
```
Comment (sonichi) [inline README.md:32]:
```suggestion
or (for users who prefer the alias pyautogen)
```

File: README.md [hunk 2]
```diff
@@ -20,25 +23,27 @@
...
+
 ```
 pip install pyautogen
 ```
+
 or
```
Comment (sonichi) [inline README.md:38]:
```suggestion
or (for users who prefer the alias autogen)
```

**Review by sonichi — COMMENTED** [review]
looks good. @qingyun-wu please check my suggestions.

File: README.md [hunk 3]
```diff
@@ -106,15 +108,17 @@ The project is currently maintained by a [dynamic group of volunteers](MAINTAINE
...
-    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/ag2ai/ag2?quickstart=1)
+   [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/ag2ai/ag2?quickstart=1)
 
- 2. Copy OAI_CONFIG_LIST_sample to ./notebook folder, name to OAI_CONFIG_LIST, and set the correct configuration.
- 3. Start playing with the notebooks!
+2. Copy OAI_CONFIG_LIST_sample to ./notebook folder, name to OAI_CONFIG_LIST, and set the correct configuration.
+3. Start playing with the notebooks!
 
-*NOTE*: OAI_CONFIG_LIST_sample lists gpt-4o as the default model. If you use a different model, you may need to revise various system prompts (especially if using weaker models like gpt-4o-mini). Proceed with caution when updating this default and be aware of additional risks related to alignment and safety.
+_NOTE_: OAI_CONFIG_LIST_sample lists gpt-4o as the default model. If you use a different model, you may need to revise various system prompts (especially if using weaker models like gpt-4o-mini). Proceed with caution when updating this default and be aware of additional risks related to alignment and safety.
```
Comment (Hk669) [inline README.md:121]: i dont think this change is required. the bold ones looks good in the readme.

**Comment by qingyun-wu** [comment]
@olaoluwasalami , downloads stats already added. Considering the conflicts in the file. I guess it is easier to create a new PR? Thank you!

**Pull request closed without merging**